sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1560,6 +1560,7 @@ private SentryOptions(final boolean empty) {
 
       eventProcessors.add(new MainEventProcessor(this));
       eventProcessors.add(new DuplicateEventDetectionEventProcessor(this));
+      eventProcessors.add(new SentryRuntimeEventProcessor());
```

Comment:
this is going to be executed by Android for every event and transaction although its never going to do anything since those properties don't exist, how can we prevent that? Android has its own Runtime, which is the ART or Dalvik

Reply:
Hmm didn't know its not there in Android. Maybe then we add a method to this processor like `isConfigured()` (or better name), that checks if these properties are set and add to event processors only then?

Reply:
to be honest I'd not add the event processor at all based on a "mode/flag" if possible, executing the event processor is already "wasting resources", maybe `SentryAndroidOptions` passes via the super ctor that its a mobile mode and don't add the event processor? we can reuse this flag for some other cases that run into the same issue, default value of this flag is obviously "true"

Reply:
I meant that based on the value returned from `isConfigured` event processor would or wouldn't be added to options. So only during the initialization the check would have been done.

Reply:
It could be an integration that when `register` is called it adds this processor to the list in the `options` it receives.

On the Android init code we just remove the integration from the list?

Reply:
could be too but sounds like overengineering having an integration per event processor for every use case like this, a boolean would do the job if its something meant for Android or not

Reply:
@maciejwalkowiak up to you either adding a new integration that gets added by `sentry` by default and removed by `sentry-android-core`, or just adding a boolean that decides if we add the event processor or not, the later sounds better IMO

Reply:
I've changed it to add event processor only when running on JVM.